src/t8_forest/t8_forest_adapt.cxx

```diff
@@ -540,12 +543,12 @@ t8_forest_adapt (t8_forest_t forest)
   /* If any subelement is constructed, give output this number as an additional information. */
   if (count_subelements > 0) {
     t8_global_productionf
-      ("Done t8_forest_adapt with %lld total elements, %lld of which are subelements.\n",
-       (long long) forest->global_num_elements, count_subelements);
+      ("Done t8_forest_adapt with %li total elements, %i of which are subelements.\n",
+       forest->global_num_elements, count_subelements);
```

Comment:
ah, so we report the global number (= sum over all processes) of elements, but only the local (= this process) number of subelements here?

Reply:
Ah yes this might be right (didnt notice the difference as I was working on only one process). Count_subelement counts all subelements that are constructed during adaptation which should refer to only one process, right? I could change global_num_elements to local_num_elements in order to align both outputs?

Reply:
Hm or a global number of subelements would be more reasonable instead but this would require an analogue function to `t8_forest_comm_global_num_elements ()`, right?

Reply:
And another question: for forest->global_num_elements is %li right or %lld with (long long) prefix?
Going to the root definition of global_num_elements it is defined as long long.

Reply:
I changed global_num_elements to local_num_elements for now and wait what you think. For me, this somehow makes sense since the global number of elements is printed in forest.c in the end of the commit function. It would just be nice to print the global number of subelements there as well.